.github/ISSUE_TEMPLATE/Standard Issue.md

```diff
@@ -0,0 +1,51 @@
+## Story
+**As a user of <STDcheck, Healthlabs, UTITreatment and/or Manage>**, I need <some functionality expressed as the user of the app would express the need at a high level>.
```

Comment:
Would the actor be more specific? e.g. Customer, Sales Agent, Patient Services Agent, Administrator?

Reply:
yes, the actor should be more specific. I am thinking the list would be <Patient, Call Center Agent, Administrator> not sure if we need to distinguish between the call center agent types, Sales, Patient Services, Chat, and Clinical Operations but will check with Shannon on that.

Reply:
I spoke with Shannon and she prefers using Agent, Admin, or Patient for the "user" field. I will make that change.